src/views/AboutUs.vue

```diff
@@ -39,6 +39,27 @@ v-container(fluid='' style='margin: 0px; padding: 0px; width: 100%;')
                   h3.font-weight-light.mb-n.pa-3(style="text-align: center;color:#000") {{item.subheading}}
               v-flex(xs12='',sm12='', md4='',lg4='',xl4='')
                 v-img.mt-5.ml-n3.mr-3(:src="item.src" contain aspect-ratio='2.5')    
+  div(align='center' justify='center')
+    v-container.col-sm-10(align='center' justify='center')
+     v-row.py-2
+      v-flex.pt-6.px-3(lg5='', justify='center')
+       v-img.img-fluid(src="../assets/contactUs.png")
+      v-flex.px-2.mx-3(justify='center' align='center')
+        div.mt-5.Form
+            iframe(name='hiddenframe' height='0' width='0' border='0' style='display: none;' onload="if(submitted){location.reload()}")
+            v-form(class="login my-3" max-height='300px' id="user_info_submit" name="form"
+                  action="https://docs.google.com/forms/u/1/d/e/1FAIpQLSd25Z-mQHDN42vNZdMXwQXkH_LfTnpUMA2kxJPnj4VHy6q8JQ/formResponse"
```

Comment:
@Sandesh1013  who owns this form?

Reply:
Currently I am the owner with Devlups given an editor access ,shall I switch the access?

Reply:
I was not able to see the sheet :(